common/pulp_docker/common/tarutils.py

```diff
@@ -114,3 +114,23 @@ def get_youngest_children(metadata):
                 pass
 
     return list(image_ids)
+
+
+def get_image_manifest(tarfile_path):
+    """
+    Given a path to a tarfile, this returns the decoded manifest.json file if it exists.
+
+    :param tarfile_path: full path to tarfile
+    :type tarfile_path:  basestring
+    :return:             decoded manifest json if it exists, otherwise None
+    :rtype               list or dict
+    """
+    image_manifest = []
```

Comment:
maybe add a comment that some of the v1 images might contain the manifest.json depending if it is v1 or v1.+